app/views/admin/tag_sets/index.html.erb

```diff
@@ -0,0 +1,26 @@
+
+<div class="page-header"><h1>Listing Tag Sets</h1></div>
+
+<table class="table">
+  <thead>
+    <th>Name</th>
+    <th>Tag group (i7)</th>
+    <th>Tag group (i5)</th>
```

Comment:
In later stories, to unify single index and dual index tag sets through the API from Limber, we will need to allow a single tag set with one tag group. I am not sure (i7) and (i5) naming would be correct in this case. @KatyTaylor Do you think this naming is OK? That is, calling the first tag group as (i7) and the second as (i5) always?

Reply:
I have sent an email about this but it does not stop getting the story to UAT.